src/style.css

```diff
@@ -0,0 +1,134 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap);
+
+* {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  box-sizing: border-box;
+  max-width: 200px;
+  background: #fff;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__top {
+  max-height: 206px;
+}
+
+.card__top-image {
+  box-sizing: border-box;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 4px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 16px 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(/images/star-active.svg);
+}
+
+.card__stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card__price-sum {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card__button-link {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border: none;
+  border-radius: 5px;
+  margin: 0 16px 16px;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover